website/docs/faqs/Accounts/delete-users.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How do I delete a user in dbt Cloud?
+description: "Deleting a user in dbt Cloud"
+sidebar_label: 'How to delete a user'
+id: delete-users
+
+---
+
+This article outlines the steps for deleting a user in dbt Cloud. If the user has a 'developer' license type, this will open up their seat for another user or allow the admins to lower the total number of seats. You must be an account owner or admin to perform these actions.
```

Comment:
```suggestion
To delete a user in dbt Cloud, you must be an account owner or have admin privileges. If the user has a `developer` license type, this will open up their seat for another user or allow the admins to lower the total number of seats. 
```

Reply:
sugg:
- conciseness
- consistency. in your file change-user-license.md file, "developer" had code font around it (back ticks vs ticks)

Reply:
Same as the above, we wanted an intro statement. If we need to trim the excess in a similar way we can dope the lines about "seats" but going for a warmer intro